src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -1145,6 +1148,7 @@ namespace OpenMS
     compound.molecular_formula = tr_it->SumFormula;
     compound.smiles_string = tr_it->SMILES;
     compound.setMetaValue("CompoundName", tr_it->CompoundName);
+    compound.setMetaValue("Adducts", tr_it->Adducts);
```

Comment:
```suggestion
    if (!tr_it->Adducts.empty()) compound.setMetaValue("Adducts", tr_it->Adducts);
```
will prevent all the empty adducts tags